windows-gaming-driver/src/driver/qemu.rs

```diff
@@ -113,7 +113,24 @@ pub fn run(cfg: &Config, tmp: &Path, data: &Path, clientpipe_path: &Path, monito
     qemu.args(&["-netdev", &usernet, "-device", "e1000,netdev=unet"]);
 
     for slot in cfg.machine.vfio_slots.iter() {
-        qemu.args(&["-device", &format!("vfio-pci,host={},multifunction=on", slot)]);
+    	let vfio_device = match slot {
+			&VfioDevice::Temporarily(ref device) => {
+				let mut child = Command::new(Path::new(::DATA_FOLDER).join("vfio-ubind")).arg(device).spawn().expect("failed to run vfio-ubind");
+				match child.wait() {
+					Ok(status) => 
+						if !status.success() {	
+						error!("vfio-ubind failed with {}! The device might not be bound to the vfio-driver and therefor not functional", status);
+						},
+					Err(err) => error!("failed to wait on child. Got: {}", err)
+					
+				}
+				device
+			}
+    		&VfioDevice::Permanent(ref device) => 
+    			device
```

Comment:
A match where both arms return the same thing generally indicates that you're doing it wrong. But this is not really relevant since that thing sholdn't be an enum to begin with.